tools/utils.nim

```diff
@@ -17,7 +17,10 @@ when not defined(vkCustomLoader):
   when defined(windows):
     const vkDLL = "vulkan-1.dll"
   elif defined(macosx):
-    const vkDLL = "libMoltenVK.dylib"
+    if defined(libMoltenVK):
```

Comment:
should be using when instead of if so it is defined correctly at compile. Yes I know this is from 2021 but I thought I should at least leave a comment if anyone ever gets back to this.

Reply:
Good point. Thanks for pointing that out. I've updated the PR.